src/components/Review/ReviewSection.tsx

```diff
@@ -1,23 +1,46 @@
 import React, { useState } from 'react'
-import { Button, Container, Grid, Header, Icon, Label, Segment, Accordion } from 'semantic-ui-react'
+import {
+  Button,
+  Card,
```

Comment:
Unused.

Reply:
It was the code removed in the conflict that used to use. Reverted now